tests/app/main/test_frameworks.py

```diff
@@ -4986,15 +5049,15 @@ def test_framework_agreement_returned_having_signed_contract_variation_redirects
     @mock.patch('dmutils.s3.S3')
```

Comment:
Looks like these 2 patches have already been added to the class, hence the failing test

Reply:
Oops - I think that's leftover from a rebase conflict.

Reply:
Now fixed (fingers crossed!)